diff --git a/setup.py b/setup.py
index 84f98f5..e898828 100755
--- a/setup.py
+++ b/setup.py
@@ -36,11 +36,10 @@ setup(
         "svgelements>=1,<2",
         "shxparser>=0.0.2",
         "Pillow>=4.0",
-        # Note there are no PyPI OpenCV packages for ARM
-        # (Raspberry PI, Orange PI, etc...)
-        "opencv-python==4.5.5.64 ; "
-        + "(\"arm\" not in platform_machine) and "
-        + "(\"aarch64\" not in platform_machine)",
+
+        # NixOS can build them, hopefully they work
+        "opencv-python>=4.12.0",
+
 	"tkinter-gl>=1.0"
     ],
     entry_points={
